### ISSUE TITLE
How to run the bot?

### ISSUE THREAD

**Issue by Tehcatninja** [Luigimaster1/embedbot#9]
Sorry for wasting your time with me being bad at things like this but, theres no real explanation on how to run the bot itself.

**Comment by HYP3RD34TH** [comment]
Yeah I wanted to add it to the `README.md` file before but i just didn't have the time for it to write a full tutorial how to run the bot... 
Nor do i think the others did... I'll or someone else will make sure they will be added soon though...
And since most of the dev's in the Team are focused towards Windows... its possible that it wont work for linux just yet... I'll try to be more active for linux fixes and such... I still need @Luigimaster1 to re-add me to the repo though...
